### ISSUE TITLE
Don't abort if gene set is not found in gmt file

### ISSUE THREAD

**Issue by mikekucera** [BaderLab/EnrichmentMapApp#205]
If a gene set name in an enrichment file is not found in the gmt file then an exception is thrown and the entire task is aborted. The error message to the user is not very helpful either. 

Note, Its pretty common for gene set names to not match up correctly because of a de-accenting bug in GSEA. When this happens the user needs to go into the enrichment file and fix it. The error message should tell them that.

When an unmatched gene set is encountered record the name of the gene set and keep going. When the task is complete report the unmatched gene sets to the user. If there are too many unmatched gene sets (ie above some reasonable threshold, like 10) then its ok to abort.

**Comment by risserlin** [comment]
Can we keep the de-accenting code in EM?  Going through the gmt file to fix all the accents is a hassle and if you are using an updated gmt file then you would have to do this every time you get a new gmt file.

Are there any issues with keeping the de-accenting code in?

**Comment by mikekucera** [comment]
It was removed before I started working on EM. I can easily add it back if you think that is the right thing to do.

**Comment by risserlin** [comment]
From going through the code it looks like it was removed around the time we went from cytoscape 2.8 to 3.0.  Why would this reappear now?

Some history:  The gene sets that used to cause the problem were from NCI-nature set and were for most part a handful of pathway that had "naive" in the pathway name.  NCI-nature database was frozen a few years ago and we are currently using a static version that I manually fixed.  From going over the bug #195 it looks like the new issue is from a Reactome pathway.

GSEA reads in the gmt file no problem but it does something to the wonky characters such that the output file no longer matches the input file.  The weird thing is that I think I was using the geneset file in the edb directory which given that it generated by gsea it should match the enrichment file result which is also generated by gsea.  

Given that this is a new instance of the characters I think EM should behave like GSEA and be able to read the geneset and match the geneset to the results even if it has non-unicode characters.
